src/hooks/useAxiosWithAuth0.js

```diff
@@ -20,15 +24,19 @@ export default function useAxiosWithAuth0() {
   };
 
   useEffect(() => {
-    (async () => {
-      const token = await getAccessTokenSilently({
-        audience: AUTH0_AUDIENCE,
-      });
-      axiosWithAuth(token);
-    })();
-  }, []);
+    if (isAuthenticated) {
+      // check if they are logged in before getAccessTokenSilently which caused errors on homepage
+      (async () => {
+        const token = await getAccessTokenSilently({
+          audience: AUTH0_AUDIENCE,
+        });
+        axiosWithAuth(token);
+      })();
+    }
+  }, [isAuthenticated]);
```

Comment:
[isAuthenticated] was added to prevent reaching for an access token before being logged in